src/dist/replication/lib/replica_stub.cpp

```diff
@@ -2258,6 +2262,85 @@ void replica_stub::open_service()
                 }
                 return result;
             });
+
+    _abnormal_read_trace_threshold_command = dsn::command_manager::instance().register_app_command(
+        {"abnormal_read_trace_threshold"},
+        "abnormal_read_trace_threshold [num | DEFAULT]",
+        "control stub log the abnormal read trace threshold",
+        [this](const std::vector<std::string> &args) {
+            std::string result("OK");
+            if (args.empty()) {
+                result = "abnormal_read_trace_threshold=" +
+                         std::to_string(_abnormal_read_trace_threshold);
+                return result;
+            }
+
+            if (args[0] == "DEFAULT") {
+                _abnormal_read_trace_threshold = _options.abnormal_read_trace_threshold;
+                return result;
+            }
+
+            int32_t threshold = 0;
+            if (!dsn::buf2int32(args[0], threshold) || threshold < 0) {
+                return std::string("ERR: invalid arguments");
+            }
+            _abnormal_read_trace_threshold = threshold;
+            return threshold == 0 ? std::string("WARN:you will close the trace log") : result;
+        });
+
+    _abnormal_write_prepare_ack_threshold_command =
+        dsn::command_manager::instance().register_app_command(
```

Comment:
it's highly repeated to write a dist commands for each config item. remove them all. i will later support one command to update all configs via flag api.

Reply:
I can't agree more.